src/native/corehost/hostpolicy/standalone/coreclr_resolver.cpp

```diff
@@ -23,6 +23,33 @@ bool coreclr_resolver_t::resolve_coreclr(const pal::string_t& libcoreclr_path, c
     coreclr_resolver_contract.coreclr_execute_assembly = reinterpret_cast<coreclr_execute_assembly_fn>(pal::get_symbol(coreclr_resolver_contract.coreclr, "coreclr_execute_assembly"));
     coreclr_resolver_contract.coreclr_create_delegate = reinterpret_cast<coreclr_create_delegate_fn>(pal::get_symbol(coreclr_resolver_contract.coreclr, "coreclr_create_delegate"));
 
+#if defined(_WIN32) && defined(_M_IX86)
+    // Some languages/toolchains (e.g. Zig) have trouble with the notion of
```

Comment:
> defined(_WIN32) &&

Is this part necessary?

Reply:
We only need the extra lookups on `win-x86` since `stdcall` isn't really a thing anywhere else.

Also, I just copied this condition from another place in the code base that deals with this `stdcall` difference as well.

Reply:
This specifically (was on mobile earlier): https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/coreclr/hosts/inc/coreclrhost.h#L11-L15